cmd/minishift/cmd/delete.go

```diff
@@ -61,6 +60,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if clearCache {
+		fmt.Printf("Deleting Minishift ISO, OpenShift container images and OpenShift origin binaries from %s/cache\n", constants.Minipath)
```

Comment:
The message should be concise (and simple): "Removed cache", or as suggested above "Removed cache at ~/.minishift/cache"